src/test/java/org/takes/rs/RsPrintTest.java

```diff
@@ -97,6 +100,33 @@ public void closeBodyEvenWhenExceptionHappens() throws IOException {
         );
     }
 
+    @Test
+    public void simple() throws IOException {
+        final StringWriter writer = new StringWriter();
+        final String firstline = "HTTP/1.1 500 Internal Server Error";
+        final Set<String> head = Collections.singleton(firstline);
+        new RsPrint(new RsSimple(head, "")).printHead(writer);
+        MatcherAssert.assertThat(
```

Comment:
@rusefillc Please use `Assertion` here, instead of static method. And `HasToString` instead of calling `toString`